src/Octoshift/Commands/ReclaimMannequin/ReclaimMannequinCommandHandler.cs

```diff
@@ -38,7 +38,6 @@ public async Task Handle(ReclaimMannequinCommandArgs args)
                 throw new OctoshiftCliException($"File {args.Csv} does not exist.");
             }
 
-            //TODO: Get verbiage approved
             if (args.SkipInvitation)
             {
                 _ = _confirmationService.AskForConfirmation("Reclaiming mannequins with the --skip-invitation option is immediate and irreversible. Are you sure you wish to continue? (y/n)");
```

Comment:
So, here's the question then: if we want the user to have a "--no prompt" option; should the default be 'Y'? Maybe we should change the option language to '--force' instead of "--no prompt"?

Reply:
In general, I actually like the idea of `--force` as the argument name, but unfortunately, we already use `--force` in this command:

> Map the user even if it was previously mapped

Given that, I think that `--no-prompt` is probably the best name.

Reply:
--no-prompt added in 36c83ba8d74077cc04000ab19e948f81fbebcab5 :tada: